cardano-cli/src/Cardano/CLI/EraBased/Commands/Governance.hs

```diff
@@ -38,7 +38,7 @@ data GovernanceCmds era
   | GovernanceDRepCmds
       (GovernanceDRepCmds era)
   | GovernancePollCmds
-      (GovernancePollCmds era)
+      GovernancePollCmds {- TODO Add era when legacy commands are removed -}
```

Comment:
The `era` type argument should stay so this command will only appear in the Babbage era.

Reply:
Done :+1: